Country-specific "Who We Are" node links

DFurnes opened the pull request:
#### Changes

Fixes #5629 by adding theme settings to set a unique node to link "Who We Are" to for each country. I've also added an update hook to automatically migrate the old settings to the new country-specific ones.
#### How should I manually test this?

Pull this branch and run the update hook. Everything should still be good. If you go to the admin panel, you should see fields to set a link for each country, pre-filled with the old universal link. If you change the link for a given country, you should see that node linked on pages with that country's prefix in the URL.
#### Screenshots

<img width="405" alt="screen shot 2015-10-27 at 10 45 57 am" src="https://cloud.githubusercontent.com/assets/583202/10762588/15e1cfd8-7c9d-11e5-9dd9-3876b3500352.png">

DFurnes committed:
Formatting and array literal syntax.

DFurnes committed:
Allow different "Who We Are" nodes per country code.

DFurnes commented:
For the most part, this seems good. The autocomplete on these theme setting fields is broken though, which makes the admin experience _frustrating_. I checked the same field on staging and it doesn't seem to autocomplete there either, so perhaps this is already a known issue. The field type is `entity_autocomplete`.

I've also experienced intermittent issues with Drupal discarding all these settings, though I can't seem to reliably reproduce it yet. Definitely not ready to merge, but would be happy to get any :eyes:s on this early to catch any obvious issues!

angaither commented:
I was testing campaign group stuff and noticed autocomplete fields were borked locally as well.

mikefantini commented:
@DFurnes since we're able to get the field to reference the proper node by putting in the full copy (i.e. It's Your World [id:1674]) I think we can ship with this as is. Obviously not an ideal long-term solution, but it works. 

Do you mind opening (or finding) an issue for autocomplete on this field so that we fix that later?

DFurnes committed:
Fix theme setting field name so it saves properly.

DFurnes commented:
Alright, this seems safe to merge now! Rebased, and fixed a wee issue that was causing Drupal to drop all the settings (2b8dc59). Will make another issue about the entity autocomplete problem.

angaither commented:
:+1:
